### ISSUE TITLE
Version mismatch for the Docker image of the Giotto viewer

### ISSUE THREAD

**Issue by zhiiiyang** [giotto-suite/Giotto#76]
Hello, thank you for contributing to such a wonderful tool. I was trying to install the Giotto viewer on my Windows computer. When I followed the instructions and downloaded the Docker image from this website (http://spatialgiotto.rc.fas.harvard.edu/giotto.install.docker.html). The downloaded file is shown as docker-giotto-4.3.7z but the version showed as 4.2 after unzipping it using 7zip (see below). 
![image](https://user-images.githubusercontent.com/7269113/95665967-01a37480-0b0a-11eb-9069-95356f1ed37f.png)
Is it okay to continue installing this docker image and use the current version of vignette on this? Just want to make sure that the workflow vignette was built under the correct version of Docker image. Thank you so much in advance.

**Comment by RubD** [comment]
Thank you for your kind words Zhi Yang. 

I'm not sure about which version is currently used for the Docker Image, but maybe @bernard2012 might help you with your questions. He created the Docker image and also the Giotto Viewer.

**Comment by bernard2012** [comment]
Thanks zhiiiyang for your interest. Maybe you can wait until later this week. We are preparing a new image containing latest Giotto Viewer and Analyzer to be released either today or tomorrow. I will let you know when that happens. Short answer to your question is that the file name is a typo, it contains version 4.3 but then it is named as 4.2. We recommend testing until the new version is released. Thanks!

**Comment by zhiiiyang** [comment]
@bernard2012 , thank you for getting back to me. I was able to get to partially host the viewer with a few edits. 
- Based on the [JSON](http://spatial.rc.fas.harvard.edu/giotto-viewer/setup.cortex.jun28.json) file provided, it turned out that the smfish_read_config function requires more arguments than what is specified in the JSON file, i.e, gene_map, dir_gene_expression, and gene_list. I went to remove those arguments from the python script and was able to host the viewer. 
- However, the map No.2 wasn't working which could be due to errors in reading in those files. I have already tried to replace two files (kmeans_annot_information.txt and cell_types_annot_information.txt) which seemed to be corrupted (?, see below). None of them worked so far. 
![image](https://user-images.githubusercontent.com/7269113/96027920-7b3ea980-0e0d-11eb-9e13-4694b8d37f84.png)

I will appreciate if you have some thoughts on the top of the head. If not, I can wait for the next release for the docker image. Thank you again for making a good tool. 

![image](https://user-images.githubusercontent.com/7269113/96027570-e89e0a80-0e0c-11eb-9897-8f8c387f14b9.png)
